Automattic-Tracks-iOS/Model/UploadError.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public enum UploadError: Error, LocalizedError {
+    case httpError(String)
+    case fileMissing
+
+    public var errorDescription: String?{
```

Comment:
In this case it does, yep, because the object conforms to `LocalizedError`, [which specifies all of its strings as optional](https://developer.apple.com/documentation/foundation/localizederror?changes=_4). 

And nice catch – we should get SwiftLint going on this repo.

Reply:
Oh I see, that's bummer. Thanks for pointing it out!